index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Eco-competition | Home</title>
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./styles/index.css" />
+    <script
+      src="https://kit.fontawesome.com/d81b9d7959.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header>
+      <div class="logo">
+        <a href="./index.html"
+          >2021 Eco <br />
+          Competition</a
+        >
+      </div>
+      <nav>
+        <i id="menu-icon" class="fas fa-bars"></i>
+        <div>
+          <ul>
+            <li><a href="./about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+          </ul>
+          <button type="button">SWM Campaign</button>
+        </div>
+      </nav>
```

Comment:
- [ ] As I said in the video the navbar should contain two columns.

- **Your design**

![Opera Instantané_2021-11-19_123805_localhost](https://user-images.githubusercontent.com/61562626/142617103-b257af66-d130-45a4-81dd-305566852aea.png)


- **Behance design**

![Capture web_19-11-2021_123833_www behance net](https://user-images.githubusercontent.com/61562626/142617195-7a4f9d46-bf0d-422b-af3b-993e6d810b30.jpeg)